src/ifp/include/ifp/InitFloorplan.hh

```diff
@@ -76,11 +77,11 @@ class InitFloorplan
                      int core_space_top,
                      int core_space_left,
                      int core_space_right,
-                     const std::string& site_name);
+                     const std::vector<odb::dbSite*>& sites);
```

Comment:
The ```sites``` arg is really extra_sites now that we auto-detect the required sites from the netlist.  If you made it
```
const std::vector<odb::dbSite*>& sites = {}
```
then you wouldn't need
```
[floorplan.findSite("FreePDK45_38x28_10R_NP_162NW_34O")]
``` 
in all the python tests